src/keycloak/chart/templates/uds-package.yaml

```diff
@@ -165,26 +165,30 @@ spec:
                 prefix: /metrics
           redirect:
             uri: "/realms/{{ .Values.realm }}/account"
+          {{- if .Values.thirdPartyIntegration.tls.tlsCertificateHeader }}
```

Comment:
Should these ones all be `if not`?

Reply:
Yes, just found it in UDS Identity Config test pipeline! Thanks for spotting this!